src/sheet_metal_conus.py

```diff
@@ -404,9 +404,8 @@ def beVerbose(self, dictCone, unitFactor, parent):
         text = inkex.etree.SubElement(parent, 'text', text_atts)
         text.text = "%4.2f deg" %(value)
         # chord lines
-        dash_style = arrow_style
+        dash_style = deepcopy(arrow_style)
```

Comment:
was drawing cone height arrow and chord arrows in wrong scale. This, and other changes in this commit, fixes that